spec/challenge.spec.js

```diff
@@ -1,127 +1,104 @@
-
-      const dequeuedPlane = this.atc.dequeue();
-      const secondDequeuedPlane = this.atc.dequeue();
-
-      expect(dequeuedPlane).to.deep.eq(this.passengerPlaneLarge);
-      expect(secondDequeuedPlane).to.deep.eq(this.passengerPlaneLarge);
-    });
-    it('dequeues large passenger planes before small passenger planes', function () {
-      this.atc.enqueue(this.passengerPlaneLarge);
-      this.atc.enqueue(this.passengerPlaneSmall);
-      this.atc.enqueue(this.passengerPlaneSmall);
-      this.atc.enqueue(this.passengerPlaneLarge);
-
-      const dequeuedPlane = this.atc.dequeue();
-      const secondDequeuedPlane = this.atc.dequeue();
-
-      expect(dequeuedPlane).to.deep.eq(this.passengerPlaneLarge);
-      expect(secondDequeuedPlane).to.deep.eq(this.passengerPlaneLarge);
-    });
-    it('dequeues large cargo planes before small cargo planes', function () {
-      this.atc.enqueue(this.cargoPlaneLarge);
-      this.atc.enqueue(this.cargoPlaneSmall);
-      this.atc.enqueue(this.cargoPlaneSmall);
-      this.atc.enqueue(this.cargoPlaneLarge);
-
-      const dequeuedPlane = this.atc.dequeue();
-      const secondDequeuedPlane = this.atc.dequeue();
-
-      expect(dequeuedPlane).to.deep.eq(this.cargoPlaneLarge);
-      expect(secondDequeuedPlane).to.deep.eq(this.cargoPlaneLarge);
-    });
-    describe('dequeues planes of equal importance in the order they were enqueued', function () {
-      beforeEach(function () {
-        this.atc = new Atc();
-
-        // add order to planes
-        this.firstCargoPlane = Object.assign(this.cargoPlaneLarge, { order: 'first' });
-        this.secondCargoPlane = Object.assign(this.cargoPlaneLarge, { order: 'second' });
-        this.firstPassengerPlane = Object.assign(this.passengerPlaneSmall, { order: 'first' });
-        this.secondPassengerPlane = Object.assign(this.passengerPlaneSmall, { order: 'second' });
-
-        this.atc.enqueue(this.firstPassengerPlane);
-        this.atc.enqueue(this.firstCargoPlane);
-        this.atc.enqueue(this.secondCargoPlane);
-        this.atc.enqueue(this.secondPassengerPlane);
-
-        this.firstDequeued = this.atc.dequeue();
-        this.secondDequeued = this.atc.dequeue();
-        this.thirdDequeued = this.atc.dequeue();
-        this.fourthDequeued = this.atc.dequeue();
-      });
-      it('dequeues the first plane correctly', function () {
-        expect(this.firstDequeued).to.deep.eq(this.firstPassengerPlane);
-      });
-      it('dequeues the second plane correctly', function () {
-        expect(this.secondDequeued).to.deep.eq(this.secondPassengerPlane);
-      });
-      it('dequeues the third plane correctly', function () {
-        expect(this.thirdDequeued).to.deep.eq(this.firstCargoPlane);
-      });
-      it('dequeues the fourth plane correctly', function () {
-        expect(this.fourthDequeued).to.deep.eq(this.secondCargoPlane);
-      });
-    });
-  });
-});
+'use strict'
+
+const chai = require('chai')
```

Comment:
I'd prefer the change to js-standard and changes to the tests to be two separate commits - it's hard to review what you changed when the whole file changed due to js-standard.